### ISSUE TITLE
DrMemory fail to start webkit test

### ISSUE THREAD

**Issue by derekbruening** [DynamoRIO/drmemory#1637]
_From [zhao...@google.com](https://code.google.com/u/106321947286816917100/) on September 22, 2014 15:10:42_

.\tools\valgrind\chrome_tests.bat -t webkit -b out\Release --tool drmemory
...
Starting 1 worker ...
[0/1] accessibility/inline-continuations.html
Failed to start the content_shell process:
content_shell took too long to startup.
[1/1] accessibility/inline-continuations.html failed unexpectedly (content_shell crashed [pid=8712])

_Original issue: http://code.google.com/p/drmemory/issues/detail?id=1637_
